$OpenBSD: patch-tools_lld_ELF_Symbols_cpp,v 1.2 2019/01/28 06:27:28 jca Exp $

Generate __data_start symbol that marks the start of .data when __data_start
is referenced from code being linked.

Index: tools/lld/ELF/Symbols.cpp
--- tools/lld/ELF/Symbols.cpp.orig
+++ tools/lld/ELF/Symbols.cpp
@@ -28,6 +28,7 @@ using namespace lld;
 using namespace lld::elf;
 
 Defined *ElfSym::Bss;
+Defined *ElfSym::Data;
 Defined *ElfSym::Etext1;
 Defined *ElfSym::Etext2;
 Defined *ElfSym::Edata1;
